hnn_core/feed.py

```diff
@@ -2,65 +2,82 @@
 
 # Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
 #          Sam Neymotin <samnemo@gmail.com>
+#          Christopher Bailey <bailey.cj@gmail.com>
 
 import numpy as np
 from neuron import h
 
 
 class ExtFeed(object):
-    """"The ExtFeed class.
+    """"The ExtFeed class of external spike input times.
+
+    An external input "feed" to the network, i.e., one that is independent of
+    the spiking output of cells in the network.
 
     Parameters
     ----------
-    ty : str
-        The feed type. Can be:
-        'extpois', 'evprox', 'evdist', 'extgauss', 'extinput'
-    celltype : str | None
-        The cell type.
-    p_ext : dict | list (XXX: check)
-        Parameters of external input.
-        p_ext has a different structure for the extinput
-        usually, p_ext is a dict of cell types
+    feed_type : str
+        The feed type, which is one of
+        'extpois' : Poisson-distributed input to proximal dendrites
+        'extgauss' : Gaussian-distributed input to proximal dendrites
+        'evprox' : Proximal input at specified time (or Gaussian spread)
+        'evdist' : Distal input at specified time (or Gaussian spread)
+        'common' : Common pulses to proximal and distal dendrites
+    cell_type : str | None
+        The cell type, e.g., 'L2_basket', 'L5_pyramidal', etc., or None for
+        common inputs (cell population-level weights for common inputs
+        are defined separately in the parameters)
+    p_ext : dict
+        Parameters of the external input feed, arranged into a dictionary.
     gid : int
         The cell ID.
 
     Attributes
     ----------
+    nrn_EventVec : NEURON hoc.HocObject
+        A vector (list) of event times
```

Comment:
this is an object from Neuron right? It's a peculiar object that's unlike a list. I would just name the actual object or remove the (list)